examples/liquidity_pool/src/main.sw

```diff
@@ -0,0 +1,42 @@
+contract;
+
+use std::{
+    address::Address,
+    context::msg_amount,
+    context::call_frames::{msg_asset_id, contract_id},
+    assert::assert,
+    contract_id::ContractId,
+    token::{mint_to_address, transfer_to_output}
+};
+
+abi LiquidityPool {
+    fn deposit(recipient: Address);
+    fn withdraw(recipient: Address);
+}
+
+const BASE_TOKEN:b256 = 0x9ae5b658754e096e4d681c548daf46354495a437cc61492599e33fc64dcdc30c;
```

Comment:
format sway files

Reply:
I did, and still got the same result on these lines @adlerjohn

Reply:
Ah, I see now, just a space between the colon and type. Got it.